src/parser/Parser.ts

```diff
@@ -830,7 +865,7 @@ export class Parser {
             }
 
             let toReturn = expression();
-            while (match(Lexeme.Newline));
+            while (match(Lexeme.Newline, Lexeme.Colon));
```

Comment:
This fixes an issue with using return in a single-line if statement (`if true: return true: end if`) and also, as a byproduct, enables support for multiple returns in a single statement (`return true : return true`)